### DISCUSSION TITLE
Change nonce to something else...

### DISCUSSION THREAD

**Discussion by Qws** [braintree/braintree_dotnet#131] [Polls]
I'm new user, I'm looking around for my app.

![image](https://user-images.githubusercontent.com/3944421/215266578-44272e95-e5ea-452f-ae28-fccb18f75843.png)

![image](https://user-images.githubusercontent.com/3944421/215266602-bac5b358-4e63-45a6-a3c7-743862674d28.png)

I keep seeing this word "nonce", in API in examples, but I don't know what it means, I haven't seen this word before, so I googled it.

But apparently:

![image](https://user-images.githubusercontent.com/3944421/215266544-5c224206-6eb8-4f64-9423-640e00921161.png)

Seriously, what does nonce means to you? Wasn't there any other possibly term you could had used?

**Comment by hollabaq86** [comment]
👋 @Qws thanks for the feedback. The term originates from the [computer security term](https://csrc.nist.gov/glossary/term/nonce), but a lot of folks who work here know that there's the other slang meaning. I don't have an ETA on if/when this term will be changed to something without this connotation, but I have forwarded the feedback on to our docs team in the hopes that we can atleast update our documentation a bit to make it more clear which `nonce` term we're referring to
